protocols/identify/src/lib.rs

```diff
@@ -35,20 +35,35 @@
 //!
 //! # Usage
 //!
-//! The [`Identify`] struct implements a `NetworkBehaviour` that negotiates
+//! The [`Behaviour`] struct implements a [`NetworkBehaviour`] that negotiates
 //! and executes the protocol on every established connection, emitting
-//! [`IdentifyEvent`]s.
+//! [`Event`]s.
 //!
 //! [Identify]: https://github.com/libp2p/specs/tree/master/identify
-//! [`Identify`]: self::Identify
-//! [`IdentifyEvent`]: self::IdentifyEvent
-//! [`IdentifyInfo`]: self::IdentifyInfo
+//! [`Behaviour`]: self::Behaviour
+//! [`Event`]: self::Event
+//! [`Info`]: self::Info
 
-pub use self::identify::{Identify, IdentifyConfig, IdentifyEvent};
-pub use self::protocol::{IdentifyInfo, UpgradeError, PROTOCOL_NAME, PUSH_PROTOCOL_NAME};
+pub use self::behaviour::{Behaviour, Config, Event};
+pub use self::protocol::{Info, UpgradeError, PROTOCOL_NAME, PUSH_PROTOCOL_NAME};
```

Comment:
Let's provide a type alias for `Info` too and then this is entirely non-breaking. That means it should actually just be a patch version bump!

Reply:
yeah awesome, it's what I going to ask about :)